bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -327,7 +327,7 @@ protected void buildAreas() {
                     String href = StringUtils.removeAll(remainingTokens[0], "\"");
                     String target = remainingTokens.length > 1 ? StringUtils.removeAll(remainingTokens[1], "\"") : "";
 
-                    Link link = linkHandler.getLink(href, target).orElse(null);
+                    Link link = linkHandler.getLink(href, target, null, null).orElse(null);
```

Comment:
this change is not needed as getLink with 2 parameters should still be supported.